### DISCUSSION TITLE
nx monorepository integration

### DISCUSSION THREAD

**Discussion by johannesschobel** [epicweb-dev/epic-stack#570] [Ideas]
Dear @kentcdodds ,

since `nx` recently released the official `@nx/remix` package with full support for remix ( 🥳 ), i think, it would be awesome to support epic-stack within a nx monorepository.

What kind of adaptations need to be done in order to get it to work? Maybe you can provide some background and assistance? That would be awesome.

Thank you very much for your time and effort!

**Comment by kentcdodds** [comment]
I think this is a great idea! I'm going to need someone put together an example. I'm afraid I don't know personally how to do this because I've not done it before, but I welcome anyone to put together an [example](https://github.com/epicweb-dev/epic-stack/blob/main/docs/examples.md)
